queries/c_sharp/highlights.scm

```diff
@@ -288,3 +286,11 @@
  "where"
 ] @keyword
 
+[
+  "return"
+] @keyword.return
+
+[
+  "yield"
+] @keyword.yield
+
```

Comment:
When capturing a single item, the recommended way is to get it directly, instead of through an alternation :
```suggestion
"return" @keyword.return

"yield" @keyword.yield
```

Reply:
fixed, thank you